Pinta.Effects/Effects/TileEffect.cs

```diff
@@ -196,5 +218,17 @@ public sealed class TileData : EffectData
 
 		[Caption ("Intensity"), MinimumValue (-20), MaximumValue (20)]
 		public int Intensity { get; set; } = 8;
+
+		[Caption ("Wave Type")]
+		public TileType WaveType { get; set; } = TileType.CleavedEdges;
 	}
 }
+
+public enum TileType
+{
+	[Caption ("Cleaved Edges")]
+	CleavedEdges,
+
+	[Caption ("Curved Edges")]
```

Comment:
How about just `Curved` or `Round`?
From what I recall when testing it out last week, in this mode it's more like the whole tile is curved, whereas the other mode is more like a flat tile except at the edges